Manage.c

```diff
@@ -1688,6 +1696,7 @@ int Manage_subdevs(char *devname, int fd,
 					close(sysfd);
 				goto abort;
 			}
+			break;
```

Comment:
this is not missing break, it is on purpose. with this break `mdadm --set-faulty` is broken.

Reply:
I removed the break;